test/web/10-sessionService.js

```diff
@@ -0,0 +1,138 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import delay from 'delay';
+import {SessionService} from 'bedrock-web-session';
+import {login, createAccount} from './helpers.js';
+import mockData from './mock-data.js';
+
+const sessionService = new SessionService();
+
+describe('sessionService API', () => {
+  describe('unauthenticated request', () => {
+    let session;
+    beforeEach(function() {
+      session = null;
+    });
+    afterEach(async function() {
+      await sessionService.logout();
+    });
+    it('should get a session with no account', async () => {
+      let err;
+      try {
+        session = await sessionService.get();
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(session);
+      session.should.be.an('object');
+      const keys = Object.keys(session);
+      // an unauthenticated session has no data
+      keys.should.deep.equal([]);
+    });
+  }); // end unauthenticated request
+  describe('authenticated request', () => {
+    let session, account, totp, email, password = null;
```

Comment:
code stye does not allow defining multiple variables on one line, please separate each def to a separate line.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-web-session/pull/9/commits/94f6ce22182e0830e69a81e9f20db591ea4337e4